src/modules/dashboard/components/ExpenditurePage/Stages/StakeExpenditureDialog.tsx

```diff
@@ -0,0 +1,162 @@
+    defaultMessage: 'Stake',
+  },
+});
+
+interface Props {
+  cancel: () => void;
+  close: () => void;
+  onClick?: () => void;
+}
+
+const StakeExpenditureDialog = ({ cancel, onClick, close }: Props) => {
+  const handleSubmit = useCallback(() => {
+    onClick?.();
+    close();
+  }, [onClick, close]);
+
+  return (
+    <Dialog cancel={cancel}>
+      <div className={styles.dialogContainer}>
+        <DialogSection appearance={{ theme: 'heading', border: 'bottom' }}>
+          <Heading
+            appearance={{ size: 'medium', margin: 'none' }}
+            className={styles.title}
+          >
+            <FormattedMessage {...MSG.header} />
+            <div className={styles.forceContainer}>
+              <FormattedMessage {...MSG.force} />
+              {/* Connect to the form in next PR */}
+              <Form initialValues={{ force: false }} onSubmit={() => {}}>
+                <div className={styles.toggleContainer}>
+                  <Toggle name="force" appearance={{ theme: 'danger' }} />
+                </div>
+              </Form>
+              <Tooltip
+                content={
+                  <div className={styles.tooltip}>
+                    <FormattedMessage id="tooltip.forceAction" />
+                  </div>
+                }
+                trigger="hover"
+                placement="top-end"
+              >
+                <Icon name="question-mark" className={styles.questionIcon} />
+              </Tooltip>
+            </div>
+          </Heading>
+        </DialogSection>
+        <DialogSection appearance={{ border: 'bottom' }}>
+          <div className={styles.stakeContainer}>
+            <FormattedMessage {...MSG.stake} />{' '}
+            <div className={styles.label}>
+              <span className={styles.icon}>
+                <TokenIcon
+                  className={styles.tokenIcon}
+                  token={activeToken}
+                  name={activeToken.name || activeToken.address}
+                />
+              </span>
+
+              <Numeral
+                unit={getTokenDecimalsWithFallback(activeToken.decimals)}
+                value={
+                  activeToken.balances[COLONY_TOTAL_BALANCE_DOMAIN_ID].amount
+                }
+              />
+              <span className={styles.symbol}>{activeToken.symbol}</span>
+            </div>
+          </div>
+        </DialogSection>
+        <DialogSection>
+          <div className={styles.messageContainer}>
+            <p className={styles.messageParagraph}>
+              <FormattedMessage {...MSG.descriptionText} />
+            </p>
+            <p className={styles.messageParagraph}>
+              <FormattedMessage {...MSG.descriptionText2} />
+            </p>
+          </div>
+        </DialogSection>
+      </div>
+      <DialogSection appearance={{ align: 'right', theme: 'footer' }}>
+        <Button
+          appearance={{ theme: 'secondary', size: 'medium' }}
```

Comment:
You implemented the design but I think it's an oversight on the product part.

@arrenv, the dialog cancel and submit buttons have smaller space between them than all other dialogs. Please, see the examples below. I think it should be the same space as everywhere in the dapp. It looks too close in my opinion anyway.
<img width="580" alt="Screenshot 2022-06-15 at 19 34 33" src="https://user-images.githubusercontent.com/34057551/173915126-95b922c9-c4c8-4080-876d-feab1098bfff.png">
<img width="714" alt="Screenshot 2022-06-15 at 19 34 16" src="https://user-images.githubusercontent.com/34057551/173915142-ff36c16b-812f-4180-b1db-6ae4a6231320.png">

By the way the margins for the title are also different than other dialogs. @arrenv, should we harmonise the dialog headings as well?

Reply:
@chinins yes I agree.